### PR TITLE
EOS-18538: BE: CSM- Audit Log: Update Audit log UI in tabular format

### PR DESCRIPTION
**Pull request by tb-s**
# Backend

## Problem Statement

[EOS-18538: BE: CSM- Audit Log: Update Audit log UI in tabular format](https://jts.seagate.com/browse/EOS-18538)

See ticket for details.

## Unit testing on RPM done

Yes

## Problem Description

See tickets for details.

### Description

- On S3 plugin, log messages with log level `info` instead of `audit`
- Convert audit log entries to serialized JSON format
- Have rsyslog parse JSON on audit log entries, and separate the log fields on Elasticsearch input payload
- Have `/api/v1/auditlogs/csm` return a list of JSON objects instead of a list of strings
- Implement paging for audit logs view

### Known issues

- In order to successfully have rsyslog parse JSON entries, [the timestamp had to be removed from cortx-utils' `cortx.utils.log` audit log config](https://github.com/Seagate/cortx-utils/tree/eos-18538--tadeu.bastos).  Please notice that this change does not affect rsyslog's ability to infer the correct timestamp, as it is added automatically to every entry by rsyslog itself; the explicit mention of a timestamp on `cortx.utils.log` is already redundant.
- Entries provided by rsyslog to Elasticsearch do not conserve the original JSON type present in the logs, e.g., numbers or `null`s are converted into strings---an empty string in case of `null`.

## Test Cases

N/A

image

### CODE REVIEW

File: csm/core/controllers/audit_log.py [hunk 1]
```diff
@@ -50,7 +54,10 @@ async def get(self):
 
         start_date = request_data["start_date"]
         end_date = request_data["end_date"] 
-        return await self._service.get_by_range(component, start_date, end_date)
+        limit = request_data.get('limit')
```
Comment (kupranay) [inline csm/core/controllers/audit_log.py:57]: fields needs to be consistent with other paging rest api of CSM

**Review by kupranay — CHANGES_REQUESTED** [review]

File: csm/core/controllers/audit_log.py (hunk 1 above)
Comment (soniyamoholkar) [reply csm/core/controllers/audit_log.py:57]: Done.

**Comment by kupranay** [comment]
run mini-provisioning

**Comment by kupranay** [comment]
run mini-provisioning

**Pull request merged**